docs/06-concepts/02-models.md

```diff
@@ -161,72 +161,98 @@ extension MyExtension on MyClass {
 |[**unique**](database/indexing)|Boolean flag to make the entries unique in the database.                                         |✅|||
 |[**default**](#default-values)|Sets the default value for both the model and the database. This keyword cannot be used with **relation**.                                                     |✅|||
 |[**defaultModel**](#default-values)|Sets the default value for the model side. This keyword cannot be used with **relation**.                                                   |✅|||
-|[**defaultDatabase**](#default-values)|Sets the default value for the database side.  This keyword cannot be used with **relation** and **!persist**.                                |✅|||
+|[**defaultPersist**](#default-values)|Sets the default value for the database side.  This keyword cannot be used with **relation** and **!persist**.                                |✅|||
 
 ## Default Values
 
-Serverpod supports defining default values for fields in your models. Following are the supported default values:
+Serverpod supports defining default values for fields in your models. The following are the supported default values:
 
 ### DateTime
 
-Supports two types of default values:
+#### Supported Default Values
 
-1. **Current Date and Time**:
-   - `now`
+| Type                       | Keyword | Description |
+|----------------------------|---------|-------------|
+| **Current Date and Time**   | `now`   | Sets the field to the current date and time. |
+| **Specific UTC DateTime**   | UTC DateTime string in the format `yyyy-MM-dd'T'HH:mm:ss.SSS'Z'` | Sets the field to a specific date and time. |
 
-   Example:
+**Example:**
 
-   ```yaml
-   dateTimeDefault: DateTime, default=now, defaultModel=now, defaultDatabase=now
-   ```
-
-2. **UTC DateTime String**:
-   - The format should be: `yyyy-MM-dd'T'HH:mm:ss.SSS'Z'`
-
-   Example:
-
-   ```yaml
-   dateTimeDefault: DateTime, default=2024-05-01T22:00:00.000Z, defaultModel=2024-05-01T22:00:00.000Z, defaultDatabase=2024-05-01T22:00:00.000Z
-   ```
+```yaml
+dateTimeDefaultNow: DateTime, default=now, defaultModel=now, defaultPersist=now
+dateTimeDefaultUtc: DateTime, default=2024-05-01T22:00:00.000Z, defaultModel=2024-05-01T22:00:00.000Z, defaultPersist=2024-05-01T22:00:00.000Z
+```
 
 ### Boolean
 
-Supports `true` or `false` values.
+#### Supported Default Values
+
+| Type                       | Keyword | Description |
+|----------------------------|---------|-------------|
+| **Boolean Values**   | `true` or `false`  | Sets the field to a boolean value, either `true` or `false`. |
 
-Example:
+**Example:**
 
 ```yaml
-boolDefault: bool, default=true, defaultModel=false, defaultDatabase=true
+boolDefault: bool, default=true, defaultModel=false, defaultPersist=true
```

Comment:
Suggestion: Move `default=true, defaultModel=false, defaultPersist=true`these variables to its own section in the top and explain the difference between them and how they interact.

Then for each type only use the `default` keyword as the example.